### PR TITLE
Readiness Probe for dashboard.

### PR DESCRIPTION
**Pull request by whenhellfreezes**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

This creates a readiness probe for dashboard.

<br/><br/>

2. What **changes to custom.yaml** are required?

None.

<br/><br/>

3. Have you documented any additional setup steps or configurations options?

NA

<br/><br/>

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Ran in minikube.

<br/><br/>

### CODE REVIEW

File: dashboard/templates/dashboard.yaml [hunk 1]
```diff
@@ -21,6 +21,13 @@ spec:
         resources:
 {{ toYaml .Values.dashboard.resources | indent 10 }}
         {{- end }}
+        readinessProbe:
```
Comment (chrisbsmith) [inline dashboard/templates/dashboard.yaml]: Should we set the same for a liveliness probe so the pod can auto restart if there's an issue?

Comment (whenhellfreezes) [reply dashboard/templates/dashboard.yaml]: I don't have a particularly strong opinion. I have been doing readiness probes for debugging feedback not service reliability. I can copy the probe if you want.

**Comment by tilleryd** [comment]
Relates to #391.

File: dashboard/templates/dashboard.yaml (hunk 1 above)
Comment (tilleryd) [reply dashboard/templates/dashboard.yaml]: What's the call here? I don't think we want to hold this PR hostage. If we can want to add liveness probes we can do so in follow up PRs.

**Comment by whenhellfreezes** [comment]
Following @tilleryd 's comment on the issue will add a liveness probe.

**Comment by whenhellfreezes** [comment]
@chrisbsmith @tilleryd liveness added.

**Comment by whenhellfreezes** [comment]
Tested and working. LGTM

**Pull request merged**